content/docs/guide-development.md

```diff
@@ -66,30 +59,65 @@ For example:
+
+Of course, relying on external data archives and data access services may pose a problem for reproducibility of results and potentially for the service operational stability. The external services may have downtime period, they may be upgraded and change their API etc. We leave it to the developer to make sure the requests to external services are operational and reproducible. We encourage the developers to supply tests that will be automatically executed from time to time during service operations and are supposed to always yield exactly the same results. If this is not so, this may signal a problem with non-availability of some external services, in this cases you, as the service developer, will be alerted and invited to investigate the origin of the problem.  It may also happen that the notebook would not produce the exactly
+the same result every time but still be reproducible (see motivation on [the difference between reproducibility and repeatability](https://github.com/volodymyrss/reproducibility-motivation/)). 
+
+ ## 2) Make the notebook available for deployment on [MMODA](https://www.astro.unige.ch/mmoda/) via [renkulab.io](https://renkulab.io/)
+
+The parameterized workflow formulated as a Python notebook can be converted into a service provided by [MMODA](https://www.astro.unige.ch/mmoda/)  by a bot that scans a specific location `astronomy/mmoda` in the project directory on the  [renkulab.io](https://renkulab.io/) collaborative platform. Creating a new project in this directory will make it visible for the bot. In our example of Fermi/LAT lightcurve workflow, it is in the [fermi](https://renkulab.io/projects/astronomy/mmoda/fermi) subdirectory of `astronomy/mmoda`.
+
+To proceed along this way, you first you need to make sure your notebook runs correctly in the  [renkulab.io](https://renkulab.io/) environment. You can start a new project in the `astronomy/mmoda` by clicking on the "Creat new project" button.
+
+![image](tmp4.png)
+
+You will need to choose the name of the new project. This name will be the name of your service appearing at the [MMODA](https://www.astro.unige.ch/mmoda) frontend and discoverable via [MMODA](https://www.astro.unige.ch/mmoda) API. Place your project in the `astronomy/mmoda` namespace by specifying this in the "Namespace" field as shown below:
```

Comment:
```suggestion
You will need to choose the name of the new project. This name will be the name of your service appearing at the [MMODA](https://www.astro.unige.ch/mmoda) frontend and discoverable via [MMODA](https://www.astro.unige.ch/mmoda) API. Place your project in the `astronomy/mmoda` namespace by specifying this in the "Namespace" field as shown below. Alternatively, a new project can be created using the following [URL](https://renkulab.io/projects/new?data=eyJ1cmwiOiJodHRwczovL2dpdGh1Yi5jb20vb2RhLWh1Yi9jb250cmlidXRlZC1wcm9qZWN0LXRlbXBsYXRlcyIsInJlZiI6Im1haW4iLCJ0ZW1wbGF0ZSI6IkN1c3RvbS9tbW9kYSJ9) which will set a predefined, MModa specific, session template, can be created after having filled in some basic information.
```

Reply:
Looks like this does not actually set a namespace.

Reply:
I am struggling in trying to set the namespace, I get a warning: `The namespace "astronomy/mmoda" is not available.` 

Any ideas? Could it be permission related?

Reply:
We had a problem with setting namespace discussed with @rokroskar before. I am not sure what's the status of this option now. If it does not work, let's avoid the link (I guess that's why I removed it eventually and left manual instructions).

Reply:
ok, I also tried to play around with the variables of the custom template, without succeeding.
I saw that I was not allowed to set any namespace different from my own username

no link then...